docs/azure/aks.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+mapt offers several operations to manage environments within azure:
+
+## AKS
+
+It creates / destroy a basic AKS Cluster. In order to run this operation the user / app requires a custom role, to create or check it see the [azure setup script](./../../hacks/azure/azure_setup.sh)
+
+### Operations
+
+#### Create
+
+This will create a windows desktop accordig to params specificed:
```

Comment:
This should be `It will crete an AKS cluster` instead? and will it create a control plane and worker nodes as well or just a control plane; would be nice to clarify how many VMs/instances will be created just to get an idea of the cose